pkg/capabilities/consensus/datafeeds/feeds_aggregator.go

```diff
@@ -0,0 +1,137 @@
+package datafeeds
+
+import (
+	"math"
+
+	"github.com/mitchellh/mapstructure"
+	"github.com/shopspring/decimal"
+	ocrcommon "github.com/smartcontractkit/libocr/commontypes"
+	"google.golang.org/protobuf/proto"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/consensus/types"
+	"github.com/smartcontractkit/chainlink-common/pkg/capabilities/mercury"
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/values"
+)
+
+type AggregatorConfig struct {
+	Tickers map[string]TickerConfig
+}
+
+type TickerConfig struct {
+	Deviation decimal.Decimal
+	Heartbeat int
+}
+
+type dataFeedsAggregator struct {
+	config       AggregatorConfig
+	mercuryCodec mercury.Codec
+	lggr         logger.Logger
+}
+
+var _ types.Aggregator = (*dataFeedsAggregator)(nil)
+
+// EncodableOutcome is a list of AggregatedPricePoints
+// Metadata is a map of feedID -> (timestamp, price) representing onchain state (see DataFeedsOutcomeMetadata proto)
+func (a *dataFeedsAggregator) Aggregate(previousOutcome *types.AggregationOutcome, observations map[ocrcommon.OracleID][]values.Value) (*types.AggregationOutcome, error) {
```

Comment:
```suggestion
func (a *dataFeedsAggregator) Aggregate(currentOutcome *types.AggregationOutcome, observations map[ocrcommon.OracleID][]values.Value) (*types.AggregationOutcome, error) {
```

Given that we plan to merge previousOutcome with newOutcome, would it make sense to call the variable `currentOutcome`?

Reply:
Not really. Aggregate takes previous outcome and returns current outcome.